### PR TITLE
Refactor all project to use alpine images, HTTPS and one docker per p…

### PR DESCRIPTION
**Pull request by ker0x**
…roject for performance

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,123 +3,113 @@
...
 
-This process assumes that depending of your OS, [Docker for Mac](https://www.docker.com/products/docker#/mac), [Docker for Windows](https://www.docker.com/products/docker#/windows) or [Docker for linux](https://www.docker.com/products/docker#/linux) is installed.
-Otherwise, [Docker Toolbox](https://www.docker.com/toolbox) should be installed before proceeding.
+This process assumes that [Docker Engine](https://www.docker.com/get-started) and [Docker Compose](https://docs.docker.com/compose/) are correctly installed.
+Otherwise, you should have a look to the [installation documentation](https://docs.docker.com/install/) before proceeding further. 
+
+### Before installation
 
-After the installation, the Proximis application is reachable by using [`http://proximis.localhost/`](http://proximis.localhost/).
+The only dependency require to run the environment is [Mutagen](https://mutagen.io/) which will synchronize your local files with docker's containers for maximize performances.
```
Comment (matthieupetel) [inline README.md]:
```suggestion
The only dependency required to run the environment is [Mutagen](https://mutagen.io/) which will synchronize your local files with docker's containers to maximize performances.
```

**Review by matthieupetel — APPROVED** [review]

File: docker-compose.yml [hunk 2]
```diff
@@ -23,33 +26,34 @@ services:
...
     volumes:
       # Projects files
-      - ${WEBSITES}:/var/www/html:rw,delegated
+      - code_sync:/var/www/html:nocopy
       # Websites configuration
-      - ./nginx/site-available:/etc/nginx/conf.d/:ro
+      - ${PROJECT_LOCATION}/docker/local/nginx.conf:/etc/nginx/conf.d/custom.conf:ro
       # Logs
-      - ./nginx/log:/var/log/nginx
+      - ${PROJECT_LOCATION}/docker/log/nginx:/var/log/nginx
```
Comment (ajardin) [inline docker-compose.yml:56]: You won't be able to extract Nginx logs by using the default image as everything is redirected to the standard output.

```
ls -l /var/log/nginx/
total 0
lrwxrwxrwx    1 root     root            11 Sep 24 23:33 access.log -> /dev/stdout
lrwxrwxrwx    1 root     root            11 Sep 24 23:33 error.log -> /dev/stderr
```

Comment (ker0x) [reply docker-compose.yml:56]: I don't know how, but logs are correctly extract on my project

Capture d’écran 2019-10-15 à 12 24 26

Comment (ajardin) [reply docker-compose.yml:56]: Indeed, it seems that now the volume declaration bypasses the redirect in the filesystem...

But even if the extract works, keep in mind that you won't be able anymore to display logs through `docker-compose logs nginx`. I don't know if it's important for you. 😉

**Review by ajardin — APPROVED** [review]

**Pull request merged**